beta/index.html

```diff
@@ -1,4 +1,12 @@
 <!--suppress HtmlUnknownTarget -->
+
+<!-- Style contributed by: TheGreatFoxxy; He tried to make it look good. -->
+<style src="/style/beta.css"></style>
+
+<div id="body">
+	
+<iframe src="/sounds/boot.wav" allow="autoplay" id="audio" style="display:none"></iframe>
```

Comment:
Probably you should use `<audio>` tag here

Reply:
gotcha, done